src/modules/menu/components/menu/feedback/FeedbackButton.vue

```diff
@@ -160,12 +160,17 @@ export default {
             }
             this.request.pending = true
             try {
+                let subject = '[web-mapviewer]'
+                if (this.feedback.rating && this.maxRating) {
+                    subject += ` [rating: ${this.feedback.rating}/${this.maxRating}]`
+                }
+                subject += ' User feedback'
                 const feedbackSentSuccessfully = await sendFeedback(
+                    subject,
                     this.feedback.message,
-                    this.feedback.rating,
-                    this.maxRating,
                     this.activeKmlLayer?.kmlFileUrl,
-                    this.feedback.email
+                    this.feedback.email,
+                    undefined // file attachment
```

Comment:
See comment above. Also personally I found better to use `null` instead of `undefined` for optional argument, but if you use the optional signature above then no need to pass anything for attachement here.